catalog/spatial/geocoding/spatial-geocoding-websearch/pom.xml

```diff
@@ -62,7 +64,9 @@
                 <configuration>
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
-                        <Embed-Dependency>json-smart,spatial4j,spatial-geocoding-api-impl</Embed-Dependency>
+                        <Embed-Dependency>
+                            json-smart,spatial4j,spatial-geocoding-api-impl,commons-lang3
```

Comment:
@coyotesqrl You're right. Just checked and these were originally on one line, so I assumed the formatter put them this way, but separating them on different lines and then running the formatter keeps them on separate lines.